### ISSUE TITLE
Troubles with code tabulations

### ISSUE THREAD

**Issue by aleksandrAndrosov** [interfaced/brightscript-intellij-plugin#14]
When you write a function, if, for etc and you finish it with end statement, no formatting of code inside of this construction happens

**Comment by aleksandrAndrosov** [comment]
also, maybe auto complete would be cool

**Comment by reeFridge** [comment]
Yeah, thanks for the report. I am trying to decide what behavior would be better
1. `End` statement auto injects after complete "head" statement (`Function foo()`, `Sub foo()` etc.)
```BrightScript
function foo() | end function
```
2. `End` statement auto injects to the next line after user typed `\n` (e.g. new line) and caret move into statement's body
```BrightScript
function foo()
->|
end function 
```

@aleksandrAndrosov What you think about this solutions?

Also autocomplete included in our road-map but with lower priority

**Comment by aleksandrAndrosov** [comment]
As for me, second behaviour is more attractive. 

There is also another part of issue "When you write a function, if, for etc and you finish it with end statement, no formatting of code inside of this construction happens"
I mean, if you were writing lots of code inside of function and it wasn't formatted, in javascript you can delete the ending "}" of function and then return it and all of the code inside turns formatted with tabulations, spaces or whatever.
